docs/source/models.rst

```diff
@@ -930,6 +938,12 @@ in the request header as shown in the record-oriented example below.
     In particular, DL models are typically strict about input types and will need model schema in order
     for the model to score correctly.
 
+    For datetime columns, values should be JSON-serialized according to ISO 8601 date or datetime format
```

Comment:
I think we should have a mini section explaining how complex data should be encoded when you have model signature. We really should have had it for binary data that is expected base64 encoded. 

so I would add something like `Encoding complex data`
And inside I would say something like:

Complex data types such as dates or binary do not native JSON representation. If you include model signature, MLflow can automatically decode supported data types from JSON. The following data type conversions are supported:
- binary: data is expected to be base64 encoded, MLflow will automatically base64decode. 
- datetime: data is expected as string according to ISO 8601 specification, for example "2021-01-01T00:00:00Z". Mlflow will parse this into appropriate  datetime representation on the given platform.

It would be cool to include an example with binary and date data.